### DISCUSSION TITLE
Generalized Bondy - Excel version

### DISCUSSION THREAD

**Discussion by hoverat** [casact/chainladder-python#477] [Q&A]
I am trying to confirm/document the generalized bondy tail.  It seems like there might be a different level of precision between the calculations in Python and Excel.  I would like to show the same CDFs between the two.  Any suggestions are appreciated.  Thank you!
 
**Python/CL code**
`triangle = cl.load_sample('tail_sample')['paid']`
`dev = cl.Development(average='simple').fit_transform(triangle)`
`tail = cl.TailBondy(earliest_age=12).fit(dev)`
`tail.cdf_`

**Excel version**
[TFWP_Appendix_August2013_CL.xlsx](https://github.com/casact/chainladder-python/files/13550896/TFWP_Appendix_August2013_CL.xlsx)

**Comment by jbogaardt** [comment]
I haven't had a chance to look too deeply yet at this and might not for a bit, but I suspect this has to do with the precision of each of the different solvers between chainladder-python (uses scipy under the hood), the TFWP paper (some older version of Excel), and your sample.  Since solvers only provide numerical approximations, I would expect different implementations to have different values within rounding of each other.  I think the only way to validate this hypothesis is to test each of the intermediate values of the overall bondy algorithm between the two.

**Reply by hoverat** [reply]
Thanks for the suggestions.  I am trying to work backwards and noticed that I am getting different ldfs for the "earliest age"
`triangle = cl.load_sample('tail_sample')['paid']`
`dev = cl.Development(average='simple').fit_transform(triangle)`
`dev.ldf_`
84-96 = 1.0466
`tail = cl.TailBondy(earliest_age=84).fit(dev)`
`tail.ldf_`
84-96 = 1.0466
`tail.earliest_ldf_`
earliest_ldf = 1.046368

It seems like these 2 numbers should be the same and I haven't been able to figure out why they are not yet.
